manifests/00-cluster-role.yaml

```diff
@@ -182,9 +182,9 @@ rules:
   - '*'
 
 - apiGroups:
-  - maistra.io
+  - sailoperator.io
```

Comment:
Does this need to be updated, or are the servicemesh3 groups in sailoperator.io?

Reply:
OSSM 3 uses the sailoperator.io API group.